tools/deb/istio-iptables.sh

```diff
@@ -319,13 +326,24 @@ if [ -n "${OUTBOUND_IP_RANGES_EXCLUDE}" ]; then
   done
 fi
 
+for internalInterface in ${REDIRECT_INTERNAL_INTERFACE}; do
```

Comment:
This block seems unnecessary since you are already inserting rule number 1 on line 338/345

Reply:
This one is necessary to create the return rules
```
Chain PREROUTING (policy ACCEPT 19 packets, 1682 bytes)
 pkts bytes target     prot opt in     out     source               destination         
   20  1742 ISTIO_REDIRECT  all  --  k6t-eth0 *       0.0.0.0/0            0.0.0.0/0           
   19  1682 RETURN     all  --  k6t-eth0 *       0.0.0.0/0            0.0.0.0/0      
```

The redirect rules came after and also insert the ISTIO_REDIRECT role so it will move it down